test/21.pooling.js

```diff
@@ -0,0 +1,171 @@
+// Copyright (c) 2018-2019, NuoDB, Inc.
```

Comment:
Will change to 2022-2023 in upcoming commit

Reply:
Just *2022* is sufficient.  We don't know if we'll work on it next year or not! :)  You don't always have to have a range, if it's not appropriate.

Reply:
If you say so.......

Reply:
I know it's pedantic, but the guideline we follow is that when you create a file you put the current year. Then you extend the range in future years iff you modify the file. The idea is that the date or date range captures the span of time over which the file is modified.

We don't, for example, update every file on Jan 01 so that the range always ends in the current year.